src/status_im/contexts/preview/quo/community/channel_actions.cljs

```diff
@@ -34,6 +34,17 @@
        :customization-color :blue
        :icon                :i/muted}]}]
    [rn/view {:height 50}]
+   [quo/channel-actions
+    {:actions
+     [{:big? true
+       :label "Pinned Messages sometimes can get very long and we have to take care of that"
+       :customization-color :blue
+       :icon :i/pin
+       :counter-value 5}
+      {:label               "Mute chat"
```

Comment:
Should not be string, but a reference to translation.

Reply:
this is a preview screen